cockroachcloud/upgrade-policy.md

```diff
@@ -39,7 +39,7 @@ If you see problems after the upgrade has been finalized, it will not be possibl
 
 ### End of Support for older CockroachDB versions
 
-As CockroachDB releases new major versions, older versions reach their End of Support (EOS) on {{ site.data.products.db }}. A CockroachDB version reaches EOS when it is 2 major versions behind the latest version. For example, now that CockroachDB v21.2 has been released, CockroachDB v20.2 has reached EOS.
+As CockroachDB releases new major versions, older versions reach their End of Support (EOS) on {{ site.data.products.db }}. A CockroachDB version reaches EOS when it is two major versions behind the latest version. For example, now that CockroachDB v21.2 has been released, CockroachDB v20.2 has reached EOS.
```

Comment:
Suggest also editing the last sentence in the paragraph to be past tense since those are pretty old versions

something like

"for example when v21.2 was released, v20.2 reached EOS"